pkg/cmd/version/version.go

```diff
@@ -19,8 +19,8 @@ type VersionInfo interface {
 func NewCmdVersion(fullName string, versionInfo VersionInfo, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "version",
-		Short: "Display client and server versions",
```

Comment:
What's wrong with the slightly longer name?

Reply:
> What's wrong with the slightly longer name?

It doesn't display server versions.